sandy/core/cov.py

```diff
@@ -812,6 +810,76 @@ def gls_update(self, S, Vy_extra, threshold=None):
         Vx_post = Vx_prior - A.dot(Vx_prior)
         return self.__class__(pd.DataFrame(Vx_post, index=index, columns=columns))
 
+    def _constrained_ls_sensitivity(self, S, threshold=None):
+        """
+        Method to obtain sensitivity according to constrained Least-Squares:
+        (S*Vx_prior*S.T)^(-1)*S*Vx_prior

```

Comment:
these are really useful! can you add these equations in all gls methods?

Reply:
to be consistent throughout the docstrings, code and reports, can you also say that M=(S*Vx_prior*S.T).
Second comment, all these equations can be written in latex style (https://stackoverflow.com/questions/10081633/using-displaymath-directives-in-docstrings-formulas). Then they would render correctly in the docs environment
Third comment, this M can be called G for analogy with the WPEC SG39 report.